templates/ui-package/cypress/support/index.js

```diff
@@ -0,0 +1 @@
+/* This file needs to exist for cypress tests to load */
```

Comment:
Why does the file need to exist even if it's empty?

Reply:
Last time I tested Cypress would crash without this file.

Reply:
Update: Looks like support folder can be deleted now